redash/query_runner/dgraph.py

```diff
@@ -93,7 +93,7 @@ def run_query(self, query, user):
         try:
             data = self.run_dgraph_query_raw(query)
 
-            first_key = next(iter(data.keys()))
+            first_key = next(iter(list(data.keys())))
```

Comment:
```suggestion
            first_key = next(data.keys())
```

Looks like the iter/list conversions are redundant.

Reply:
Apparently `iter` is needed.